ext/php7/circuit_breaker.c

```diff
@@ -69,28 +69,28 @@ static uint64_t current_timestamp_monotonic_usec() {
     return t.tv_sec * 1000 * 1000 + t.tv_nsec / 1000;
 }
 
-static int64_t get_max_consecutive_failures(TSRMLS_D) {
+static int64_t get_max_consecutive_failures() {
```

Comment:
Should we replace `TSRMLS_D` with `void`?

Reply:
I have no opinion on this. Can do that if you like.

Reply:
I think I would prefer having these be `void` since it is technically an API change to not have it.

Reply:
Will do then.